data/api/build.gradle

```diff
@@ -18,7 +18,7 @@ kotlin {
     sourceSets {
         commonMain.dependencies {
             api Dep.Kotlin.stdlibCommon
-            api project(":model")
+            implementation project(":model")
```

Comment:
`api` configuration propagates declarations to followed projects.  
It causes compile error in `:frontend:ios-combined` because `srcDirs` and dependencies have same class declarations.

It have no problem to change `api` to `implementation`.